### PR TITLE
Add phone to registration

### PR DESCRIPTION
**Pull request by MS-SAP**

### CODE REVIEW

**Review by lsc0498 — COMMENTED** [review]
Very good work!🎉

File: web/controllers/registrationController/index.ts [hunk 1]
```diff
@@ -52,6 +52,7 @@ export async function postTutorHandler(req: Request, res: Response) {
         if (typeof req.body.firstname == 'string' &&
             typeof req.body.lastname == 'string' &&
             typeof req.body.email == 'string' &&
+            typeof req.body.phone == 'string' || typeof req.body.phone == 'undefined' &&
```
Comment (Jonasdoubleyou) [inline web/controllers/registrationController/index.ts:55]: this won't work due to operator predescendence. It reads as (firstname && lastname && email && phone) || (phone && ..). You might want to put this or in braces.

File: web/controllers/registrationController/index.ts [hunk 2]
```diff
@@ -323,6 +335,7 @@ export async function postTuteeHandler(req: Request, res: Response) {
...
```
Comment (Jonasdoubleyou) [inline web/controllers/registrationController/index.ts:338]: same here

File: web/controllers/registrationController/index.ts [hunk 3]
```diff
@@ -604,6 +628,7 @@ export async function postMentorHandler(req: Request, res: Response) {
...
```
Comment (Jonasdoubleyou) [inline web/controllers/registrationController/index.ts:631]: same here

File: web/controllers/registrationController/index.ts [hunk 4]
```diff
@@ -753,6 +788,7 @@ export async function postStateTuteeHandler(req: Request, res: Response) {
...
```
Comment (Jonasdoubleyou) [inline web/controllers/registrationController/index.ts:791]: same here

**Review by Jonasdoubleyou — CHANGES_REQUESTED** [review]

File: web/controllers/registrationController/index.ts (hunk 1 above)
Comment (MS-SAP) [reply web/controllers/registrationController/index.ts]: Thanks for alerting. Fixed now.

**Review by Jonasdoubleyou — APPROVED** [review]

**Pull request closed without merging**